db/migrate/20161212205412_add_display_name_to_tutorials.rb

```diff
@@ -0,0 +1,9 @@
+class AddDisplayNameToTutorials < ActiveRecord::Migration
+  def change
+    add_column :tutorials, :display_name, :text, default: ""
+    Tutorial.find_each do |tut|
```

Comment:
For what it's worth, when doing a bulk update like this you can use the `update_all` method to avoid the batch / validations, round trips, etc `Tutorial.update_all(display_name: "")`